test/test.vim

```diff
@@ -360,6 +370,26 @@ function s:suite.pkg_setup()
 EOL
   call s:assert.isdirectory(s:optdir . '/vim-searchx')
   call s:assert.notfilereadable(s:optdir . '/_/plugin/searchx.vim')
-  call s:assert.true(jetpack#load('vim-searchx'))
-  call s:assert.equals(g:searchx.auto_accept, v:true) " Default is v:false, so if v:true, setup has been called.
+  call s:assert.true(jetpack#load('vim-searchx'), 'vim-searchx cannot be loaded')
+  call s:assert.true(g:searchx.auto_accept) " Default is v:false, so if v:true, setup has been called.
+endfunction
+
+function! s:suite.pkg_requires() abort
+  lua <<EOL
+  packer_setup({
+      'hrsh7th/nvim-cmp',
+      opt = true,
+    }, {
+      'hrsh7th/cmp-buffer',
+      requires = 'nvim-cmp',
+      opt = true,
+    }
+  )
+EOL
+  call s:assert.isdirectory(s:optdir . '/nvim-cmp')
+  call s:assert.isdirectory(s:optdir . '/cmp-buffer')
+  call s:assert.notfilereadable(s:optdir . '/_/plugin/cmp.lua')
+  call s:assert.true(jetpack#load('cmp-buffer'), 'cmp-buffer cannot be loaded')
+  call s:assert.true(jetpack#tap('nvim-cmp'), 'nvim-cmp is not loaded') " means nvim-cmp is also loaded
+  call s:assert.loaded('cmp_buffer') " means cmp-buffer/after/plugin is sourced
 endfunction
```

Comment:
The functionality of requires is not in vim-plug, and I did this because I thought it would be handled only in packer notation due to compatibility issues.
Do you want to mention in the documentation that it is not exclusively for neovim?

Reply:
Hmm. This is just my preference. Excuse me, I should not force you so to do.